package.json

```diff
@@ -997,6 +1002,11 @@
           "when": "viewItem =~ /^(gists|starredGists|followedUser).gist/",
           "group": "base@2"
         },
+        {
+          "command": "gistpad.openGistNewWindow",
+          "when": "viewItem =~ /^(gists|starredGists|followedUser).gist/",
+          "group": "base@2"
```

Comment:
For cleanliness sake, can we make this `base@3` since the expected behavior is for it to appear below the previous commands.

Reply:
Done :)